### PR TITLE
Rewrite "Search preview" guide

### LINKED ISSUE #3046: Rewrite "Search preview" article
Meilisearch Cloud features a new search preview with a revised UI. This is something the marketing team wants to emphasise in their communication.

The current article on search preview is not particularly helpful and fairly long. We should rewrite it both to make it more useful and in preparation to a possible influx of visitors.

The new article must:
- differentiate between localhost and Cloud versions of the preview
- emphasise the Cloud search preview provides a better experience
- update relevant screenshots

Note: although it is possible Meilisearch will eventually port the Cloud preview to self-hosted instances, there's not timeline for it yet.

### PR DESCRIPTION
**Pull request by guimachiavelli**
Closes #3046, #3051

### CODE REVIEW

**Review by gmourier — APPROVED** [review]

File: assets/images/cloud-getting-started/12-project-page-url.png
Comment (nicolasvienot) [inline assets/images/cloud-getting-started/12-project-page-url.png:1]: The new search preview does not contain the `NewTab` icon, but not a priority at all.

File: assets/images/search_preview/07-sidebar-options-export.png
Comment (nicolasvienot) [inline assets/images/search_preview/07-sidebar-options-export.png:1]: For this screen, you could only have this, and add a border around the download button.
You could also add a screenshot of the opened Modal, but again not a priority.

Screenshot 2024-11-21 at 13 26 04
Screenshot 2024-11-21 at 13 26 37

File: learn/getting_started/search_preview.mdx [hunk 1]
```diff
@@ -5,24 +5,44 @@ description: Meilisearch comes with a built-in search interface for quick testin
...
-![Animated gif in which a user types "trumen show" and sees "The Truman Show" as the top result](/assets/images/search_preview/trumen.gif)
+![The same search preview interface as in the previous image, but with the "Ranking score" option turned on. Search results are the same, but include the document's ranking score](/assets/images/search_preview/04-score.png)
+
+## Configuring search options
+
+Use the menu on the left-hand side to configure [sorting](/learn/filtering_and_sorting/sort_search_results) and [filtering](/learn/filtering_and_sorting/filter_search_results). These require you to first edit your index's sortable and filterable attributes. You may additionally configure any filterable attributes as facets. In this example, "Genres" is one of the configured facets:
+
+![The sidebar of the search preview interface, with a handful of options, including "Sort by", "AI-powered search", "Filters", and "Genres"](/assets/images/search_preview/05-sidebar-options.png)
+
+You can also perform [AI-powered searches](/learn/ai_powered_search/getting_started_with_ai_search) if this functionality has been enabled for your project.
```
Comment (nicolasvienot) [inline learn/getting_started/search_preview.mdx:38]: You could add here a screenshot of the AISearch component
Screenshot 2024-11-21 at 13 29 47

**Review by nicolasvienot — COMMENTED** [review]
It is also possible to `add`, `copy`, `update` and `delete` documents from the interface. Not sure it we want to add this or not.

Screenshot 2024-11-21 at 13 33 19
Screenshot 2024-11-21 at 13 33 22
Screenshot 2024-11-21 at 13 33 45
Screenshot 2024-11-21 at 13 33 57

Otherwise, it looks good to me, thanks @guimachiavelli!

File: assets/images/search_preview/06-sidebar-options-advanced.png
Comment (nicolasvienot) [inline assets/images/search_preview/06-sidebar-options-advanced.png:1]: Same for this one, I think you can crop the image and now show the full sidebar.

**Comment by guimachiavelli** [comment]
Thanks for the feedback, @nicolasvienot! 🙇 

I'd rather focus on the use case of "tweaking search settings" at the moment to keep things simple, so I will refrain from mentioning the edit/etc buttons. It seems clearer and closer to the way we expect users to interact with the preview at the moment.

It's outside the scope of this PR, but recently we started implementing functionality that allows for directly editing a database on Meilisearch (these buttons, the experimental `document/edit` route). Although I understand this can be occasionally useful, it sort of goes against our recommended best practice of "do not use Meilisearch as your primary database". I'll bring it up with product.

**Pull request merged**